tests/analysis_test.py

```diff
@@ -60,3 +65,16 @@ def get_cmd_output(self, exec_name, options=''):
                              options, self.trace_writer.trace_root)
 
         return subprocess.getoutput(cmd)
+
+    def get_output(self, name, output):
+        out = open(os.path.join(self.trace_writer.trace_root, name), "w")
```

Comment:
I'd probably use the `with`  statement here, to avoid raising a possible `OSError` exception.